### ISSUE TITLE
The root of each identifier type shuld be `PID@hostname`

### ISSUE THREAD

**Issue by acaproni** [IntegratedAlarmSystem-Group/ias#98]
The philosophy of the Identifier and the fullRunningId is to easily understand what runs where and the processing steps of each IAS type.
On top of the hierarchy there is always a process running on a server so we should redefine the [IdentifierType](https://github.com/IntegratedAlarmSystem-Group/ias/blob/develop/BasicTypes/src/java/org/eso/ias/types/IdentifierType.java) adding the new root type. All the types that have an empty ancestor should have the new type instead.

The IAS shall provide a API to build the root type by getting the PID and the host and returning the proper ID.

**Comment by acaproni** [comment]
At the present there are 3 tools that wil be affected by this change:
* CONVERTER
* PLUGIN
* SUPERVISOR

The case of the IASIO suggests that we need to think a little bit better on it. From the fullRunning of an IASIO we want to be able to understand who produced it:
* `MONITORED_SYSTEM :arrow_right: PLUGIN :arrow_right: CONVERTER :arrow_right: IASIO`
* `SUPERVISOR :arrow_right: DASU :arrow_right: ASCE :arrow_right: IASIO`

For this we defined in IdentifierType the possible linking of Identifiers that is a recursive a-cyclic data structure.
This defintion does not work well if the standalone tools like the plugin or the converter has PID@localhost as their root.

We should probably distinguish the identifiers of tools from thosee of IASIOs.

**Comment by acaproni** [comment]
To fix this ticket, we can add a new IdentifierType ROOT whose value PID@localhost is defined only at runtime.
Then the converter can have 2 ancenstors:
* `PLUGIN`: for the `IASIO`s
* `ROOT`: for the converter tool

Up to now tools like the converter and the WebServerSender do not have a Identifier and that's why we never met this problem. I found it by chance adding the heartbeat to the Converter.

In short, the Identifier as it is defined today is oriented to IASIOs and does what we need. But it does not support stand alone tools.

**Comment by acaproni** [comment]
@see #145
